Gemfile

```diff
@@ -13,16 +13,16 @@ group :development do
 end
 
 group :development, :test do
-  gem "capybara", "~> 2.6"
-  gem "coveralls", "~> 0.7", require: false
-  gem "faker", "~> 1.2"
-  gem "pry-byebug", "~> 1.2"
-  gem "rack-test", "~> 1.1"
-  gem "rspec", "~> 3.4"
-  gem "rspec-html-matchers", "~> 0.7"
-  gem "shotgun", "~> 0.9"
+  gem "capybara"
+  gem "coveralls", "> 0.8", require: false
```

Comment:
Can we not remove the version constraint here, too?

Reply:
No, because there is some recoverable conflict in dependencies. If it's not specified to be greater than 0.8 it installs a 0.7.2 (if I recall correctly) from 2014, and it doesn't support all the features implemented in the code base.

It's selected (or so I believe) because that was the last version that doesn't have a dependency on an older version of another gem we use.